student.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'person'
+
+# student.rb
+class Student < Person
+  def initialize(id, age, classroom, parent_permission: true, name: 'Unknown')
+    super(id, age, parent_permission: parent_permission, name: name)
+    @classroom = classroom
+  end
+
```

Comment:
- [ ] Please make sure to not pass id in the super constructor as it's generating in the parent class itself and pass the parameters in the sequence `super(name: name, age, parent_permission: parent_permission)` as expected in the parent class.